ftrackhq/integrations
# feat: Backlog/framework refactor/photoshop standalone application cep base

Resolves : 

* CLICKUP-https://app.clickup.com/t/865d3yjwx
* FTRACK-
* SENTRY-
* ZENDESK-


This PR has been tested on :



## Changes

- Base standalone QApplication for standalone framework process.
- Base CEP plugin
- Connection establishment.

## Test

1. Build and install CEP plugin and Connect plugin
2. Launch Photoshop, a message box should appear that integration has loaded.
3. Standalone process should log that it has connected to Photoshop.
4. Try closing panel in Photoshop, it the standalone process should stay alive.
5. Open panel again, standalone should re-connect accordingly.
6. Close Photoshop, standalone should detect this after a while and shut down.
